cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -174,7 +177,8 @@ public static String loadClassName(LoadDetail loadDetail) {
         return "EnergyConsumer";
     }
 
-    public static int getTerminalSide(Terminal t, Connectable<?> c) {
```

Comment:
Can you keep this method as deprecated?

Reply:
fixed. Method kept as deprecated and for removal